### PR TITLE
Clarify buildifier integration support

### PR DESCRIPTION
**Pull request by limdor**
Add a note that the integration of buildifier as a rule is only supported on Linux.
More information https://github.com/bazelbuild/buildtools/issues/346 and https://github.com/bazelbuild/buildtools/issues/770

### CODE REVIEW

File: buildifier/README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ See also the [full list](../WARNINGS.md) or the supported warnings.
 
 ## Setup and usage via Bazel
 
-You can also invoke buildifier via the Bazel rule.
+On Linux you can also invoke buildifier via the Bazel rule.
```
Comment (keith) [inline buildifier/README.md:83]: This rule also supports macOS so can you add that here I guess? Or just add `(not supported on Windows)` instead?

Comment (limdor) [reply buildifier/README.md]: @keith fixed

**Comment by limdor** [comment]
@keith any chance that we can get this merged? Is something missing?

**Review by vladmos — APPROVED** [review]
Thanks, sorry for the late review.

**Pull request merged**